### DISCUSSION TITLE
Section Groups Accordions – Entries Sidebar

### DISCUSSION THREAD

**Discussion by alexanderloewe** [craftcms/cms#11340] [Ideas] [answered]
In our projects we often have 20 or more sections. We arrange these in groups (with titles), but it would be great if this groups or titles could be collapsible like an accordion.

Something like this:

<img width="276" alt="Bildschirmfoto 2022-05-27 um 20 13 18" src="https://user-images.githubusercontent.com/69263307/170768180-32364c93-314e-40b6-b32e-660bc59a987c.png">

<img width="506" alt="Bildschirmfoto 2022-05-27 um 20 12 54" src="https://user-images.githubusercontent.com/69263307/170768433-114e291b-3765-4274-ae49-43d22e3f93da.png">

**Comment by siebird** [comment]
I have a few projects that could use something like this. 

Thinking the customize source modal could have an option in the heading tab to toggle accordions for all children grouped with it. This would clean up the sidebar a bunch with sites that have a ton of channels. 

/cc not sure if this was the thread you were thinking of @brandonkelly 🧐

**Comment by elfacht** [comment]
With the new [Entrification plan](https://craftcms.com/blog/entrification) this feature would be almost mandatory, as I addressed it in the duplicate  #14621

**Reply by thupsi** [reply]
I have made a similar suggestion for organizing sections: Essentialy instead of having collapsible section groups in the sections sidebar, we could have separate [section groups in the global sidebar](https://github.com/craftcms/cms/discussions/14609).

Why not have both of course 🙂

**Reply by wmdhosting** [reply]
+1

**Comment by alexanderloewe** [comment]
Any update on that one?

**Comment by brandonkelly** [comment, marked as answer]
Craft 5.8.0 is out with collapsible source headings (#17226).

**Reply by brandonkelly** [reply]
We took this a step further in Craft 5.9.0 with the ability to create custom entry index pages (#17779)
